test/razorIntegrationTests/hover.integration.test.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as jestLib from '@jest/globals';
+import testAssetWorkspace from './testAssets/testAssetWorkspace';
+import * as integrationHelpers from '../integrationTests/integrationHelpers';
+
+jestLib.describe(`Razor Hover ${testAssetWorkspace.description}`, function () {
```

Comment:
The `jestLib` is no longer needed - you can just import the functions normally now that we've removed mocha
```
import { describe, beforeAll, beforeEach, afterAll, test, expect } from '@jest/globals';
```